src/main/java/edu/unc/lib/boxc/migration/cdm/services/CdmIndexService.java

```diff
@@ -310,19 +305,15 @@ private void indexObject(Connection conn, List<String> exportFieldValues)
 
     /**
      * Create the index database with all cdm and migration fields
-     * @param force
+     * @param options
      * @throws IOException
      */
-    public void createDatabase(boolean force, CdmIndexOptions options) throws IOException {
-        ensureDatabaseState(force);
-        List<String> exportFields;
+    public void createDatabase(CdmIndexOptions options) throws IOException {
+        ensureDatabaseState(options.getForce());
 
-        if (options.getCsvFile() != null) {
-            CdmFieldInfo fieldInfo = fieldService.loadFromCsvFieldsFromProject(project);
-            exportFields = fieldInfo.listAllExportFields();
-        } else {
-            CdmFieldInfo fieldInfo = fieldService.loadFieldsFromProject(project);
-            exportFields = fieldInfo.listAllExportFields();
+        CdmFieldInfo fieldInfo = fieldService.loadFieldsFromProject(project);
+        List<String> exportFields = fieldInfo.listAllExportFields();
+        if (options.getCsvFile() == null) {
```

Comment:
I think we can remove this conditional, I think we want to have MIGRATION_FIELDS exist for CSV projects, some of the fields are used for features like group mapping.